py_zipkin/encoding/_encoders.py

```diff
@@ -176,9 +189,22 @@ def encode_queue(self, queue):
 class _V1JSONEncoder(_BaseJSONEncoder):
     """JSON encoder for V1 spans."""
 
-    def encode_span(self, span_builder):
+    def encode_remote_endpoint(self, remote_endpoint, kind, binary_annotations):
+        json_remote_endpoint = self._create_json_endpoint(remote_endpoint, True)
+        if kind == Kind.CLIENT:
+            key = 'sa'
+        elif kind == Kind.SERVER:
+            key = 'ca'
+
+        binary_annotations.append({
```

Comment:
if this is v2 json I think the format should be a plain boolean true. in thrift it is type boolean value a single bit (byte with value 1)

Reply:
In both V1 and V2 the value should be a string according to the spec. This is V1 JSON btw.